src/components/stepper/Stepper.tsx

```diff
@@ -0,0 +1,67 @@
+import StepperCircle from "./StepperCircle"
+import { MainContainer } from "./Stepper.styles";
+import { createRef, useRef, useState } from "react";
+import { Button, ButtonsContainer, StepperContainer, FormContainer } from "./StepperHandler.styles";
+
+type Step = {
+    title: string;
+    component: React.ForwardRefExoticComponent<any>;
+    props?: any;
+}
+
+
+interface StepperProps {
+    steps: Step[];
+}
+
+
+function Stepper({ steps }: StepperProps) {
+
+    const [currentStep, setCurrentStep] = useState<number>(0);
+    const formRefs = useRef(steps.map(() => createRef<any>()));
+
+    const nextStep = async () => {
+        const currentFormRef2 = formRefs.current[currentStep];
+        currentFormRef2.current.submit().then((res: any) => {
+            if (res) {
+                setCurrentStep((prev) => Math.min(prev + 1, steps.length - 1));
+            }
+        });
+    };
+
+    const backStep = () => {
+        setCurrentStep((prev) => Math.max(prev - 1, 1)); 
+    };
```

Comment:
Hay un bugsito con esto, cuando estamos en el paso 0 (se muestra el botón cancelar) en vez de reducir un paso deberiamos "cancelar la transacción". A mi parecer el Stepper deberia tener recibir un paremetro `onCancel` que sea un metodo que se llame cuando se hace click en el boton cancelar.


```suggestion
    const backStep = () => {
        currentStep === 0 ? onCancel() : setCurrentStep(prev => prev - 1) 
    };
```

Reply:
Yo lo resolví así, no sé si está mal
```suggestion
    const backStep = () => {
        if (currentStep === 0) {
            Navigate("/home")
        }
        else {
            setCurrentStep((prev) => Math.max(prev - 1, 0));
        }
    };
```

Reply:
No está mal que uses un if, es lo mismo, lo que esa mal es que el Stepper siempre te mande al `/home`. No sabés dónde se usa este componente, y no necesariamente siempre te tiene que llevar al home. Por eso el componente deberia recibir como parámetro cuál es el comportamiento que se desea hacer cuando se cancela